arm-sql/2014-04-01/swagger/serverCommunicationLinks.json

```diff
@@ -0,0 +1,308 @@
+            "required": true,
+            "type": "string",
+            "description": "The name of the server communication link."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ServerCommunicationLink"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ServerCommunicationLinks"
+        ],
+        "operationId": "CommunicationLinks_CreateOrUpdate",
+        "description": "Creates a server communication link.",
+        "x-ms-examples": {
+          "Create a server communication link": { "$ref": "../examples/ServerCommunicationLinkCreate.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "communicationLinkName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the server communication link."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServerCommunicationLink"
+            },
+            "description": "The required parameters for creating a server communication link."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ServerCommunicationLink"
+            }
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/communicationLinks": {
+      "get": {
+        "tags": [
+          "ServerCommunicationLinks"
+        ],
+        "operationId": "CommunicationLinks_ListByServer",
+        "description": "Gets a list of server communication links.",
+        "x-ms-examples": {
+          "List server communication links": { "$ref": "../examples/ServerCommunicationLinkList.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition.",
```

Comment:
nit: make descriptions more meaningful, eg.: "Communication links successfully returned", applicable to all other descriptions too